images/win/toolsets/toolset-2016.json

```diff
@@ -243,6 +245,40 @@
                     "llvm",
                     "toolchain"
                 ]
+            },
+            {
+                "arch": "mingw-w64-ucrt-x86_64",
+                "exec_dir": "ucrt64",
+                "runtime_packages": [
+                    {
+                        "name": "clang",
+                        "executables": [
+                            "clang",
+                            "g++",
+                            "gcc"
+                        ]
+                    },
+                    {
+                        "name": "cmake",
+                        "executables": [
+                            "cmake",
+                            "cpack",
+                            "ctest"
+                        ]
+                    },
+                    {
+                        "name": "ragel",
+                        "executables": [
+                            "ragel"
+                        ]
```

Comment:
@LeonidLapshin , I don't think that we should install any additional MSYS packages to Hosted images.

According to https://github.com/actions/virtual-environments/issues/1572 
> As a result, the most obvious enhancement would be NOT to install any additional packages in this environment. Just extract the latest stable release, and optionally update the environment once. Then, setup-msys2 would take care of caching (ad-hoc for each user).

cc: @AlenaSviridenko

Reply:
kk, removed

Reply:
@maxim-lobanov @MSP-Greg there is no sense in this PR at all if can't add the packages

Reply:
My first message was about not adding any new MSYS packages at all. I remember that we had a long discussion with @eine in past in https://github.com/actions/virtual-environments/issues/1525 and https://github.com/actions/virtual-environments/issues/1572 . If I remember correctly, the main point of that discussion was to not install any MSYS packages and provide pure MSYS without any pre-installed packages at all.
Obviously, we couldn't do it because it would break some customers but I guess we don't want to install more packages.